src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -3201,5 +3201,56 @@ public class C
 ";
             CreateCompilationWithMscorlibAndSpan(text).VerifyDiagnostics();
         }
+
+        [Fact]
+        [WorkItem(24776, "https://github.com/dotnet/roslyn/issues/24776")]
+        public void PointerElementAccess_RefStructPointer()
```

Comment:
>void [](start = 15, length = 4)

It would be nice to write these tests so that there are *no* errors (because sometimes errors cause suppression of other errors).

Reply:
@gafter the tests are asserting that we *can* produce these errors. The compiler crashes because binding reaches a state where it wasn't thought possible (because of erroneous nodes).